sdk/provisioning/Azure.Provisioning.CloudMachine/src/Client/MessagingServices.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Messaging.ServiceBus;
+
+namespace Azure.CloudMachine;
+
+public static class MessagingServices
+{
+    public static void Send(this CloudMachineClient cm, object serializable)
```

Comment:
Are we not able to make this async (all of the SB library methods are async)?

Reply:
the CM APIs are meant to seamlessly integrate. All APIs will be sync and async, and messaging just async? That throws a wrench into the "seamless" integration

Reply:
Yes, the AMQP libraries are just async.

Reply:
yeah, I did not mean to put a question mark. I know they are only async. I do want sync (and async APIs) here and I was thinking about adding some fire-and-forget send APIs. But as you observed, I then need to add something to report errors